### PR TITLE
🛠 domain change for my presences

### PR DESCRIPTION
**Pull request by eggsywashere**
urGENT, couldn't change the service names because if I did, it'll make me lose all current users and they'll stuck with an outdated eggsy.codes presence.

### CODE REVIEW

File: websites/E/eggsy.codes/dist/metadata.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
-    "tr": "eggsy.codes, EGGSY'nin PreMiD ile ve bir çok başka projesiyle alakalı işlerini sergilediği internet sitesidir!"
+    "en": "eggsy.xyz is where EGGSY (PV and TM) hosts his projects such as PreMiD releated presence usage stats, metadata creator and much more!",
+    "tr": "eggsy.xyz, EGGSY'nin PreMiD ile ve bir çok başka projesiyle alakalı işlerini sergilediği internet sitesidir!"
   },
-  "url": "eggsy.codes",
-  "regExp": "eggsy[.]codes(?![/]projects[/]premid[/]custom-status)",
-  "version": "1.2.3",
+  "url": "eggsy.xyz",
+  "regExp": "eggsy[.]xyz(?![/]projects[/]premid[/]custom-status)",
+  "version": "1.2.5",
```
Comment (doomlerd) [inline websites/E/eggsy.codes/dist/metadata.json:13]: .4

Comment (eggsywashere) [reply websites/E/eggsy.codes/dist/metadata.json:13]: god no

Comment (eggsywashere) [reply websites/E/eggsy.codes/dist/metadata.json:13]: no doom. no.

**Comment by eggsywashere** [comment]
> .codes was better

I like my 40 bucks better :p

**Pull request merged**